pnpm-lock.yaml

```diff
@@ -18481,6 +18564,13 @@ packages:
     dependencies:
       brace-expansion: 2.0.1
 
+  /minimatch@5.1.6:
```

Comment:
A run of `pnpm dedupe` looks like it'll cut down on some of the duplication here. In this case it'll use the existing version, while for picocolors and svgo it'll bump indirect deps elsewhere to the newer versions.